source/src/ble/cs_CrownstoneCentral.cpp

```diff
@@ -0,0 +1,663 @@
+/*
+ * Author: Crownstone Team
+ * Copyright: Crownstone (https://crownstone.rocks)
+ * Date: Apr 16, 2021
+ * License: LGPLv3+, Apache License 2.0, and/or MIT (triple-licensed)
+ */
+
+#include <ble/cs_CrownstoneCentral.h>
+#include <ble/cs_BleCentral.h>
+#include <ble/cs_UUID.h>
+#include <cfg/cs_UuidConfig.h>
+#include <encryption/cs_ConnectionEncryption.h>
+#include <encryption/cs_KeysAndAccess.h>
+
+#include <structs/cs_ControlPacketAccessor.h>
+#include <structs/cs_ResultPacketAccessor.h>
+#include <structs/buffer/cs_CharacteristicBuffer.h>
+#include <structs/buffer/cs_CharacteristicReadBuffer.h>
+#include <structs/buffer/cs_CharacteristicWriteBuffer.h>
+#include <structs/buffer/cs_EncryptionBuffer.h>
+
+#define LOGCsCentralInfo LOGi
+#define LOGCsCentralDebug LOGv
```

Comment:
reduce logging before merge with master :)

Reply:
Ah, set to LOGv for reason mentioned earlier.
The info should remain LOGi, in my opinion. I defined it so it's easy to temp disable by a developer.